Assets/Scripts/CTF/FlagCatcher.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FlagCatcher : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
```

Comment:
Just for the sake of sanitisation, I think we should remove methods that aren't used. All the typical Monobehavior methods can be added when required anyway. Makes it cleaner also :)